Libplanet.Tests/KeyStore/Kdfs/ScryptTest.cs

```diff
@@ -0,0 +1,286 @@
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using Libplanet.KeyStore;
+using Libplanet.KeyStore.Kdfs;
+using Org.BouncyCastle.Crypto.Digests;
```

Comment:
This doesn't seem to be used.

Reply:
Oh! I missed this when I purged <T> (Sha256Digest).
Is there any linter or vscode extension for preventing this anymore?

Reply:
I hope we have a such lint if there is anything. 🤔